Add flameshot as a screenshot tool option

tonyd33 opened the pull request:
Flameshot is a popular and well-featured tool used in X so it's worth mentioning in the FAQ on screenshot tools.

Although it doesn't work out of the box on Wayland, a few windowrules can be added to fix the incompatibility which have been added to the FAQ.

vaxerski commented:
idk, it's very broken atm and I wouldn't recommend it

tonyd33 commented:
> idk, it's very broken atm and I wouldn't recommend it

How so? It's working pretty much fine for me on v12.1.0 (14a13677) from aur/flameshot-git with the settings I gave.

The exception I just realized was that copying to clipboard can give warped results, but it seems to be fixed by calling `flameshot gui --raw | wl-copy` instead of just `flameshot gui`.

tonyd33 commented:
I guess you can look at it that way.

I think it's a shame to not include because the other options require much more tinkering to get features on-par with flameshot, which just requires a little configuration that'd be documented in the FAQ to work.

We could move flameshot to the last option and add emphasize the disclaimer that flameshot isn't working as intended on Wayland but the configuration fixes it or we could close this PR until flameshot implements the fixes themselves if we still think it isn't worth using. Your call

vaxerski commented:
> We could move flameshot to the last option and add emphasize the disclaimer that flameshot isn't working as intended on Wayland but the configuration fixes it

that's fine. Also worth noting that on some setups (hidpi / multimon  etc) I've heard very mixed results with flameshot

tonyd33 commented:
> Also worth noting that on some setups (hidpi / multimon etc) I've heard very mixed results with flameshot

Yeah, I'm on a HiDPI and multi-monitor setup, and I think I've come across why they have mixed results. The configuration is what solved it for me though.

I moved the Flameshot option down and added more emphasis that it's broken and users should use it at their own discretion.
